lib/cover_my_meds/api_request.rb

```diff
@@ -25,8 +25,14 @@ def request(http_method, host, path, params={}, auth_type = :basic, &block)
       rest_resource = RestClient::Resource.new(uri.to_s, tail)
 
       response = call_api http_method, rest_resource, &block
+      parse_response response
```

Comment:
I see what your getting at, but we want the response method to return nil if the response body is empty - nil is essentially the parsed response. If we only parse the response when the body is present we don't get the correct response on empty returns.